rocAL_pybind/CMakeLists.txt

```diff
@@ -19,9 +19,13 @@
 # THE SOFTWARE.
 
 cmake_minimum_required(VERSION 3.5)
-project(rocAL_pybind)
+set(VERSION "2.0.0")
+
+# Set Project Version and Language
+project(rocAL_pybind VERSION ${VERSION} LANGUAGES CXX)
 
 set(TARGET_NAME rocAL_pybind)
+option(BUILD_PYPACKAGE  "Build rocAL Python Package" ON)
```

Comment:
@kiritigowda Do not add this back and hard code it here. it needs to come from the main CMakeLists. If user doesn't want Python package, they need an option to turn it off